pages/index.tsx

```diff
@@ -1,116 +1,8 @@
-import { Box, Button, Grid, Typography } from '@mui/material';
-import {
-  School as SchoolIcon,
-  Lightbulb as LightbulbIcon,
-} from '@mui/icons-material';
 import Head from 'next/head';
-import { useContext, useState, useRef, useCallback } from 'react';
-import { useRouter } from 'next/router';
-import { throttle } from 'lodash-es';
-
-import Link from '@components/shared/Link';
 import Layout from '@components/shared/Layout';
-import { getClassroom, sampleClassroomName } from '@utils/classrooms';
-import { SocketContext } from '@contexts/SocketContext';
-import { UserContext } from '@contexts/UserContext';
-import Chatbox from '@components/pages/TeachersPage/Chatbox';
-import BasicModal from '@components/shared/Modal';
-import ModalTextField from '@components/shared/ModalTextField';
-
-const exampleChat = {
-  chatId: 'homepage sample chat',
-  studentPair: [
-    {
-      socketId: '343d11sf',
-      realName: 'Sam Carlome',
-      character: 'Perfectionist Dentist',
-    },
-    {
-      socketId: 'as31afsf',
-      realName: 'Jessica Placard',
-      character: 'Party Planner',
-    },
-  ],
-  conversation: [
-    [
-      'student1',
-      'Perfectionist Dentist',
-      "I'm finally graduating dental school!!!",
-    ],
-    ['student2', 'Party Planner', 'This calls for a party!!!!!'],
-    ['student1', 'Perfectionist Dentist', 'YESSS!!'],
-    ['student1', 'Perfectionist Dentist', 'I am so excited!'],
-    ['student2', 'Party Planner', 'I can bring party food'],
-    ['student2', 'Party Planner', 'pretzels and chocolate cake'],
-    ['student1', 'Perfectionist Dentist', 'NO!!'],
-    ['student1', 'Perfectionist Dentist', "those aren't healthy"],
-  ],
-  startTime: '',
-};
+import HomePage from '../src/components/pages/HomePage/index';
```

Comment:
This shortened version works because we defined the `/components` path in tsconfig.json in paths field. The advantage of this is if we ever move this into a different folder, the import will still work perfectly fine. 
```suggestion
import HomePage from '@components/pages/HomePage/index';
```